sound/soc/codecs/rt715-sdca.h

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: GPL-2.0 
+// 
+// rt715-sdca.h -- RT715 ALSA SoC audio driver header
+// 
+// Copyright(c) 2019 Realtek Semiconductor Corp.
+// 
+
+#ifndef __RT715_SDCA_H__
+#define __RT715_SDCA_H__
+
+#include <linux/regulator/consumer.h>
+
+struct rt715_priv {
+	struct regmap *regmap;
+	struct regmap *sdw_regmap;
+	struct snd_soc_codec *codec;
+	struct sdw_slave *slave;
+	struct delayed_work adc_mute_work;
+	int dbg_nid;
+	int dbg_vid;
+	int dbg_payload;
+	enum sdw_slave_status status;
+	struct sdw_bus_params params;
+	bool hw_init;
+	bool first_init;
+	int l_is_unmute;
+	int r_is_unmute;
+	int hw_sdw_ver;
+};
+
+struct sdw_stream_data {
```

Comment:
don't think it's a good idea to define a struct with a generic name in a codec-specific header.